edm4hep.yaml

```diff
@@ -159,8 +159,7 @@ datatypes :
       static const int BITStopped = 24 ;    \n
       static const int BITOverlay = 23 ;    \n
       /// return energy computed from momentum and mass \n
-      double getEnergy() const { return sqrt( getMomentum()[0]*getMomentum()[0]+getMomentum()[1]*getMomentum()[1]+\n
-                                        getMomentum()[2]*getMomentum()[2] + getMass()*getMass()  )  ;} \n
+      double getEnergy() const { return std::hypot(getMomentum()[0], getMomentum()[1], getMomentum()[2], getMass()) ; } \n
```

Comment:
There doesn't seem to be a 4 argument overload for `std::hypot`

Reply:
Well shoot